docs/releases/1.10.txt

```diff
@@ -36,6 +36,9 @@ Minor features
   link <django.contrib.admin.AdminSite.site_url>` at the top of each admin page
   will now point to ``request.META['SCRIPT_NAME']`` if set, instead of ``/``.
 
+* On a ``ModelAdmin``'s changelist, if a user has no add permission the
```

Comment:
You need to use a right quote for "``'s" otherwise Sphinx will render the single quote in the incorrect direction. I'd suggest simply to say "On a model's changelist" to avoid it.
"has no add" -> "doesn't have the add permission, "